test/helpers/kubeclient_helper.rb

```diff
@@ -2,23 +2,29 @@ module KubeclientHelper
   MINIKUBE_CONTEXT = "minikube".freeze
 
   def kubeclient
-    @kubeclient ||= begin
-      config = Kubeclient::Config.read(ENV["KUBECONFIG"])
-      unless config.contexts.include?(MINIKUBE_CONTEXT)
-        raise "`#{MINIKUBE_CONTEXT}` context must be configured in your KUBECONFIG (#{ENV["KUBECONFIG"]}). Please see the README."
-      end
-      minikube = config.context(MINIKUBE_CONTEXT)
+    @kubeclient ||= build_kube_client("v1")
+  end
+
+  def v1beta1_kubeclient
```

Comment:
That sounds weird to me tbh, 'cause it's the target api not the client that is v1/v1beta1. How would you feel about adding the prefix to `v1_kubeclient` instead?